Sources/TuistGenerator/Graph/PackageNode.swift

```diff
@@ -2,36 +2,29 @@ import Basic
 import Foundation
 import TuistCore
 
+class PackageDependencyNode: GraphNode {
```

Comment:
Maybe `PackageProductNode`? Now it's kind of misleading of what's the difference between `PackageNode` and `PackageDependencyNode`

Reply:
I like this, I rushed this name and knew it was not quite right.